src/views/Admin/AdminSessionDetail.vue

```diff
@@ -154,6 +163,16 @@ export default {
       body: { session }
     } = await NetworkService.adminGetSession(this.$route.params.sessionId);
     this.session = session;
+
+    // Set quill document after the DOM has been updated to show session div
+    this.$nextTick(() => {
+      if (this.session.type === "college") {
```

Comment:
Instead of checking for the session type, how about doing the same check as the template's `v-if`, i.e. just checking for the presence of `session.quillDoc`

Reply:
Great point!